src/Component/CreateAllGroupsButton/CreateAllGroupsButton.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import {
+  cleanup,
+  fireEvent,
+  render,
+  screen,
+  waitForElementToBeRemoved
+} from '@testing-library/react';
+
+import SHOGunAPIClient from '@terrestris/shogun-util/dist/service/SHOGunAPIClient';
+
+import { CreateAllGroupsButton } from './CreateAllGroupsButton';
+import GroupService from '@terrestris/shogun-util/dist/service/GroupService';
+import Group from '@terrestris/shogun-util/dist/model/Group';
+
+import type { PartialOmit } from '../../test-util';
+
+const mockService: Partial<GroupService<Group>> = {
+  createAllFromProvider: jest.fn()
+};
+
+const mockSHOGunAPIClient: PartialOmit<SHOGunAPIClient, 'group'> = {
+  group: jest.fn().mockReturnValue(mockService)
+};
+
+jest.mock('../../Hooks/useSHOGunAPIClient', () => {
+  const originalModule = jest.requireActual('../../Hooks/useSHOGunAPIClient');
+  return {
+    __esModule: true,
+    ...originalModule,
+    default: jest.fn(() => mockSHOGunAPIClient)
+  };
+});
+
+describe('<CreateAllUsersButton />', () => {
```

Comment:
```suggestion
describe('<CreateAllGroupsButton />', () => {
```

Reply:
Fixed in 38d4309ef04c93a5bd9dfc5fc663e76cfa8a3399.